### PR TITLE
Add Python any article

### PR DESCRIPTION
**Pull request by alayek**
related to `any()` in #819

### CODE REVIEW

File: python-any.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Python any(iterable)
```
Comment (koustuvsinha) [inline python-any.md]: # Python `any(iterable)`

File: python-any.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+print(any([])) #=> False
+print(any({})) #=> False
+print(any([6, 7])) #=> True
+print(any([6, 7, None])) #=> True
+print(any([0, 6, 7])) #=> True
+print(any([9, 8, [1, 2]])) #=> True
+```
+:rocket: [REPL It!](https://repl.it/CL9c/0)
+
+[Documentation](https://docs.python.org/3/library/functions.html#any)
```
Comment (koustuvsinha) [inline python-any.md]: ~~Documentation~~ Official Docs

**Comment by koustuvsinha** [comment]
# LGTM :+1:

**Comment by theflametrooper** [comment]
Thanks @alayek :shipit:

**Comment by Rafase282** [comment]
The filename is lowercaser

**Pull request merged**